fun/starboard.go.tmpl

```diff
@@ -1,24 +1,33 @@
 {{/*
 	This command allows users to react to messages with stars. If it reaches a given amount, it will be sent in a given channel.
 	Benefits over star command provided in docs: Updates star count with more stars using a single DB text entry.
-
+	Posts automatically removed when they fall below star threshold. Ability to use "anti-star" reactions similar to an
+	upvote/downvote system to automatically remove posts unfit for starboard. 
+	
 	Recommended trigger: Reaction trigger with option `Added + Removed reactions`.
 */}}
 
 {{/* CONFIGURATION VALUES START */}}
 {{ $starEmoji := "⭐" }} {{/* Star emoji name */}}
-{{ $starLimit := 1 }} {{/* Reactions needed for message to be put on starboard */}}
+{{ $starLimit := 4 }} {{/* Reactions needed for message to be put on starboard */}}
 {{ $starboard := 678379546218594304 }} {{/* ID of starboard channel */}}
 {{ $maxAge := "2w" }} {{/*maximum age of message for stars to be counted. structure (mo)nth, (w)eek, (d)ay, (h)our. ex: 3d = 3days, 1mo = 1 month.*/}}
+
+{{ $antiStarEnable := true}} {{/*enable/disable anti-star counting. enable = true, disable = false*/}}
```

Comment:
```suggestion
{{ $antiStarEnable := false}} {{/*enable/disable anti-star counting. enable = true, disable = false*/}}
```
Should be default deactivated, otherwise it might cause some confusion why "the starboard broke" (people can't read docs).